app/src/test/java/com/kickstarter/viewmodels/CreatorDashboardReferrerBreakdownHolderViewModelTest.java

```diff
@@ -0,0 +1,369 @@
+  }
+
+  @Test
+  public void testCustomReferrerColor() {
+    final Project project = ProjectFactory.project();
+    final ProjectStatsEnvelope statsEnvelope = ProjectStatsEnvelopeFactory.projectStatsEnvelope();
+
+    setUpEnvironment(environment());
+    this.vm.inputs.projectAndStatsInput(Pair.create(project, statsEnvelope));
+    this.customReferrerColor.assertValues(ReferrerColor.CUSTOM.getReferrerColor());
+  }
+
+  @Test
+  public void testCustomReferrerPercent() {
+    final Project project = ProjectFactory.project();
+    final ProjectStatsEnvelope.ReferrerStats fiftyFivePercentCustomReferrer = ProjectStatsEnvelopeFactory.ReferrerStatsFactory.referrerStats().toBuilder()
+      .percentageOfDollars(55f)
+      .referrerType(ReferrerType.CUSTOM.getReferrerType())
+      .build();
+    final List<ProjectStatsEnvelope.ReferrerStats> referrerList = Arrays.asList(
+      fiftyFivePercentCustomReferrer);
+    final ProjectStatsEnvelope statsEnvelope = ProjectStatsEnvelopeFactory.projectStatsEnvelope().toBuilder().referralDistribution(referrerList).build();
+
+    setUpEnvironment(environment());
+    this.vm.inputs.projectAndStatsInput(Pair.create(project, statsEnvelope));
+    this.customReferrerPercent.assertValues(55f);
+  }
+
+  @Test
+  public void testCustomReferrerPercentText() {
+    final Project project = ProjectFactory.project();
+    final ProjectStatsEnvelope.ReferrerStats fiftyFivePercentCustomReferrer = ProjectStatsEnvelopeFactory.ReferrerStatsFactory.referrerStats().toBuilder()
+      .percentageOfDollars(55f)
+      .referrerType(ReferrerType.CUSTOM.getReferrerType())
+      .build();
+    final List<ProjectStatsEnvelope.ReferrerStats> referrerList = Arrays.asList(
+      fiftyFivePercentCustomReferrer);
+    final ProjectStatsEnvelope statsEnvelope = ProjectStatsEnvelopeFactory.projectStatsEnvelope().toBuilder().referralDistribution(referrerList).build();
+
+    setUpEnvironment(environment());
+    this.vm.inputs.projectAndStatsInput(Pair.create(project, statsEnvelope));
+    this.customReferrerPercentText.assertValues(NumberUtils.flooredPercentage(55f * 100f));
+  }
+
+  @Test
+  public void testCustomReferrerPledgedAmount() {
+    final Project project = ProjectFactory.project();
+    final ProjectStatsEnvelope.ReferrerStats fiftyFivePercentCustomReferrer = ProjectStatsEnvelopeFactory.ReferrerStatsFactory.referrerStats().toBuilder()
+      .percentageOfDollars(55f)
+      .referrerType(ReferrerType.CUSTOM.getReferrerType())
+      .build();
+    final List<ProjectStatsEnvelope.ReferrerStats> referrerList = Arrays.asList(
+      fiftyFivePercentCustomReferrer);
+    final ProjectStatsEnvelope statsEnvelope = ProjectStatsEnvelopeFactory.projectStatsEnvelope().toBuilder().referralDistribution(referrerList).build();
+
+    setUpEnvironment(environment());
+    this.vm.inputs.projectAndStatsInput(Pair.create(project, statsEnvelope));
+    this.customReferrerPercentText.assertValues(NumberUtils.flooredPercentage(55f * 100f));
```

Comment:
should this be testing the pledged amount rather than percent text again?

Reply:
ah yep bad copypaste

Reply:
also this is an unnecessary output so just going to remove from the viewmodel